### DISCUSSION TITLE
How to install asammdf without c-extensions?

### DISCUSSION THREAD

**Discussion by PrathameshNimkar** [danielhrisca/asammdf#838] [Q&A]
I am trying to install the asammdf python package without c-extensions on py3.8 on macos using pip.
I tried to install using the following:
```
pip uninstall asammdf
pip cache purge
pip install asammdf --no-binary :all:
```

And when I checked the site packages, I still found blocks/cutils.c. 
This is used as a dependency in mdf_v4.py and mdf_v4.py and so on. 
Looking for advice on how to get rid of all c-extensions and have no .c files.
Essentially I'm looking for a pure python package to use with mf4 files.

**Comment by danielhrisca** [comment]
The C extension is mandatory
